media/nls/src/c_28604.txt

```diff
@@ -1,4 +1,4 @@
-﻿CODEPAGE 28604
+CODEPAGE 28604
```

Comment:
What's the difference there? Hidden BOM?

Reply:
Yes.